### DISCUSSION TITLE
O65 and creating a new calendar event.

### DISCUSSION THREAD

**Discussion by ghost** [O365/python-o365#894] [General]
Greetings,

Using O365 to create calendar events and all is okay so far. However I would like to add at least one custom property to each event, all with a different values. Microsoft docs mentions the so called  "singleValueExtendedProperties". 

Is this possible in O365? And if yes  how? Hope to hear, Aad Slingerland

**Comment by alejcas** [comment]
This should be posible to implement although I haven’t read the MS Graph regarding this.

Anyway this is not implemented at the moment. PR are welcome

**Reply by ghost** [reply]
Hello janscas,  Thanks for the reply. But BTW what is a PR ¯\_(⊙_ʖ⊙)_/¯

**Reply by BlueSideStrongSide** [reply]
PR is short for  "Pull Request"

That is where you would write code to add the  new/improved functionality and then submit that "code" to the maintainer for review/approval/addition to the main application.
